### ISSUE TITLE
Insufficient contrast image viewer zoom slider in forced colors mode

### ISSUE THREAD

**Issue by corrideat** [okTurtles/group-income#2385]
### Problem

Insufficient contrast / invisible zoom slider on image viewer when using [forced colors mode](https://drafts.csswg.org/css-color-adjust-1/#forced-colors-mode).

![image](https://github.com/user-attachments/assets/7b2ad0d5-f5d4-4154-abe5-7523280c943f)

### Solution

Detect forced colors and make the slider visible
